x/evm/types/params.go

```diff
@@ -281,13 +281,6 @@ func (m Params) Validate() error {
 		return err
 	}
 
-	// ensure that the network is one of the supported ones
-	for _, n := range m.Networks {
-		if n.Name == m.Network {
-			return nil
-		}
-	}
-
 	if err := validateBytes(m.GatewayCode); err != nil {
```

Comment:
Validate bytecode first

Reply:
Why? what bug did this fix?

Reply:
On devnet, I used the old bytecode params and it doesn't include `absorber`. So `add-chain` succeeded, but `VoteConfirmChainRequest` fails when setting the param set. `add-chain` should also validate the bytecode properly.

Reply:
The bug is that if the network is found in the networks list, we return `nil` right away instead of validating the bytecode.

Reply:
ok then the fix should be that if the network isn't in the list, return an error, otherwise continue instead of return. Simply reordering would leave the problem there when someone wants to add more validation.

Reply:
Sure, I prefer that anyways. Still getting used to the preferred coding style here lol